githublib.py

```diff
@@ -0,0 +1,58 @@
+#!/usr/bin/env python3
+
+import requests
+import json
+import time
+from datetime import datetime
+
+from common import get_envvar
+
+SECONDS_PER_WEEK = 604800
+
+
+class Github:
```

Comment:
After reading what this class do, I realized that 2 out of its 3 methods are about a single issue.
I would consider creating an issue class as well.